artifactory_test.go

```diff
@@ -794,11 +794,41 @@ func TestArtifactoryDownloadAndSyncDeletes(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
-	tests.CheckSyncedDirContent(tests.GetSyncExpectedDeletesDownloadStep7(), paths, t)
+	checkSyncedDirContent(tests.GetSyncExpectedDeletesDownloadStep7(), paths, t)
 
 	cleanArtifactoryTest()
 }
 
+// After syncDeletes we must make sure that the content of the synced directory contains the last operation result only.
+// Therefore we verify that there are no other files in the synced directory, other than the list of the expected files.
+func checkSyncedDirContent(expected, actual []string, t *testing.T) {
+	// Check if all expected files are actually exist
+	tests.IsExistLocally(expected, actual, t)
+	// Check if all the existing files were expected
+	err := isExclusivelyExistLocally(expected, actual, t)
+	if err != nil {
+		t.Error(err.Error())
+	}
+}
+
+// Check if only the files we were expect exist locally, i.e return an error if there is a local file we didn't expect.
```

Comment:
...only the files we were expect exist locally... --> ...only the files we expect, exist locally...